### PR TITLE
refactor(Apply): rename to rxApply

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-1146

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM (same IA hierarchy, just renamed)
- [x] QE LGTM (no tests present)

### Summary

* rename `Utilities > Apply` to `Utilities > rxApply`
* leave `Apply` alias for backward compatibility (documented as deprecated)

### CODE REVIEW

**Comment by CITguy** [comment]
### Demo
before | after
----- | -----
apply-before | rxapply-after

### Demo Nav
before | after
----- | -----
screen shot 2016-11-15 at 6 04 37 pm | screen shot 2016-11-15 at 6 03 59 pm

### ngDocs
page | before | after
----- | ----- | -----
Apply | ngdocs-apply-before | ngdocs-apply-after
rxApply | N/A | ngdocs-rxapply-after

File: src/utilities/rxApply/scripts/rxApply.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    /**
+     * @deprecated
+     * Use rxApply instead. This filter will be removed on the 4.0.0 release.
+     * @ngdoc filter
+     * @name utilities.filter:Apply
+     * @requires utilities.filter:rxApply
+     */
+    function ApplyFilter ($filter) {
+        return $filter('rxApply');
```
Comment (CITguy) [inline src/utilities/rxApply/scripts/rxApply.js]: Add `console.warn()` with depreation message.

**Comment by parlarjb** [comment]
Dev LGTM

**Comment by halkon** [comment]
LGTM

**Pull request merged**